webrender/src/renderer.rs

```diff
@@ -1384,13 +1384,30 @@ impl<B: hal::Backend> Renderer<B> {
 
         let debug_server = DebugServer::new(api_tx.clone());
 
+        #[cfg(not(feature = "gleam"))]
+        let heaps_config = match options.memory_config {
+            Some(ref path) => {
+                use std::fs::File;
+                use std::io::prelude::*;
+                use ron::de::from_str;
+                let mut file = File::open(path).expect("Unable to open memory config file");
+                let mut source = String::new();
+                file.read_to_string(&mut source).unwrap();
```

Comment:
this can be done simpler - https://doc.rust-lang.org/stable/std/fs/fn.read_to_string.html

Reply:
Addressed in 047f31bd64bae048f54c9c5902a003efae16d7dd